baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/AssertNoArgsTest.java

```diff
@@ -0,0 +1,202 @@
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a, Arg<?> arg) {",
+                        "    a.hasArgs(arg);",
+                        "  }",
+                        "}")
+                .doTest();
+    }
+
+    @Test
+    void testLoggableExceptionAssert_hasExactlyArgs() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a) {",
+                        "    a.hasExactlyArgs();",
+                        "  }",
+                        "}")
+                .addOutputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a) {",
+                        "    a.hasNoArgs();",
+                        "  }",
+                        "}")
+                .doTest();
+    }
+
+    @Test
+    void testLoggableExceptionAssert_hasExactlyArgs_notEmpty() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.Arg;",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a, Arg<?> arg) {",
+                        "    a.hasExactlyArgs(arg);",
+                        "  }",
+                        "}")
+                .addOutputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.Arg;",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a, Arg<?> arg) {",
+                        "    a.hasExactlyArgs(arg);",
+                        "  }",
+                        "}")
+                .doTest();
+    }
+
+    @Test
+    void testLoggableExceptionAssert_containsArgs() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a) {",
+                        "    a.containsArgs();",
+                        "  }",
+                        "}")
+                .addOutputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.testing.LoggableExceptionAssert;",
+                        "public class Test {",
+                        "  void f(LoggableExceptionAssert a) {",
+                        "    a.hasNoArgs();",
```

Comment:
I think this one could produce failures, but I can't imagine anyone would write `.containsArgs()` and expect it to be a no-op.

Reply:
Searching our internal repos, I only see 2 uses of `.containsArgs()` without any arguments and they are both in a repo I own. And the intention in both cases was to assert that the exception has no args.

It's probably more likely that this change is a bug fix.

Reply:
Yep :+1: